src/scss/_mixins.scss

```diff
@@ -80,4 +80,20 @@
   font-family: font(secondary);
   text-transform: uppercase;
   text-align: center;
+}
+
+@mixin footer-title {
+  @include title-font; 
+  text-align: left;
+  color: $white;
+  font-size: 0.9rem;
+}
+
+@mixin footer-text {
+  color: $silver-chalice;
+  font-size: 0.6rem;
+  text-transform: uppercase;
+  font-family: font(primary-light);
+  line-height: 1.2rem;
+  text-decoration: none;
```

Comment:
If this is only for Footer,  than whats the point of those mixins?

Reply:
it's not only for footer, i should rename it

Reply:
I plan to reuse them, so I will rename them)